### PR TITLE
Enhancement blog page

### PR DESCRIPTION
**Pull request by DoozyDoz**
# Changes
- added a button to the blog page to create a new blog
- changed the appearance of blog items in the blog page
- added functionality to edit an existing blog via the "edit" button in the blog details page
- added a mongoose [server](https://github.com/CirroCloudLtd/Server) to the serve the blogs

# Files changed
- blog.js
- blog.html
- blog-form.html
- blog-details.html

# Screenshots
Before Change 1 Screenshot
Before Change 1 Screenshot
Before Change 1 Screenshot
Before Change 1 Screenshot

### CODE REVIEW

**Comment by sempsmahad** [comment]
The ui looks fake and u have conflicts boss

File: js/blog.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
+const blogApp = {
+  blogList: [],
+  currentPage: 0,
+  itemsPerPage: 4,
+
+  fetchBlogList: async function () {
+    try {
+      const response = await fetch('http://localhost:5000/api/v1/blogs');
```
Comment (sempsmahad) [inline js/blog.js:8]: use our [server](server.cirrocloudug.com) inside of ur local host

File: js/blog.js [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+    content.appendChild(readMoreButton);
+
+    info.appendChild(header);
+    info.appendChild(underline);
+    info.appendChild(content);
+
+    const blogDetails = document.createElement('div');
+    blogDetails.classList.add('blog-details');
+
+    // const title = document.createElement('h2');
```
Comment (sempsmahad) [inline js/blog.js:90]: remove comments before commiting

File: js/blog.js [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+          content: item.querySelector('h5').innerText,
+        };
+        localStorage.setItem('selectedBlog', JSON.stringify(blogData));
+        window.location.href = 'blog-details.html';
+      });
+    });
+  },
+
+  updatePaginationButtons: function () {
+    // const prevBtn = document.querySelector('.prev-btn');
```
Comment (sempsmahad) [inline js/blog.js:156]: remove

**Comment by DoozyDoz** [comment]
@sempsmahad I already merged the changes so i will just have to fix these issues in a fresh PR

**Pull request merged**